types/gulp-pug-linter/gulp-pug-linter-tests.ts

```diff
@@ -0,0 +1,8 @@
+import gulp = require('gulp');
+import * as gulpPugLinter from "gulp-pug-linter";
+
+gulp.task('lint:template', () => (
+    gulp
+        .src('./**/*.pug')
+        .pipe(gulpPugLinter({ reporter: 'default' }))
+));
```

Comment:
Since this file is called "tests" (plural) and the `reporter` can be either a string, like you're testing here with `'default'` or a function, do you think it's worth adding another test to verify that?

Reply:
@ilyakam, I added other test in last commit.